Modix.Data/Models/Moderation/NoteEntity.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Modix.Data.Models.Moderation
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+
+    public class NoteEntity
+    {
+        [Key, Required, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+        public ulong UserId { get; set; }
+        public string UserName { get; set; }
+        public string Message { get; set; }
+        public DateTime RecordedTime { get; set; }
```

Comment:
Should be `DateTimeOffset`.

Reply:
Also, for consistency with other entitites, I'd recommend `Created` and `CreatedBy` as property names.